app/components/course-page/introduction-step/create-repository-card.ts

```diff
@@ -15,6 +15,7 @@ interface Signature {
 
   Args: {
     repository: RepositoryModel;
+    preferredLanguageSlug: string | undefined;
```

Comment:
We usually sort args alphabetically btw (helps avoid ambiguity, especially with longer lists of args)